dockerize.go

```diff
@@ -230,7 +232,37 @@ func main() {
 			}
 			template, dest = parts[0], parts[1]
 		}
-		generateFile(template, dest)
+
+		fi, err := os.Stat(template)
+		if err != nil {
+			log.Fatalf("unable to stat %s, error: %s", template, err)
+		}
+		if fi.IsDir() {
+			if dest != "" {
```

Comment:
Could you move this whole block L241-L262 to a separate func like `generateDir`?